akka-actor-typed-tests/src/test/scala/docs/akka/typed/RouterSpec.scala

```diff
@@ -143,5 +143,58 @@ class RouterSpec extends ScalaTestWithActorTestKit("akka.loglevel=warning") with
 
       probe.receiveMessages(10)
     }
+    "show group routing with consistent hashing" in {
+
+      val probe1 = createTestProbe[String]()
+      val probe2 = createTestProbe[String]()
+
+      object Proxy {
+
+        val RegisteringKey = ServiceKey[Command]("aggregator-key")
+
+        def mapping(command: Command) = command.id
+
+        sealed trait Command {
+          def id: String
+        }
+        case class Message(id: String, content: String) extends Command
```

Comment:
same here, only `Message` will do the trick.

Reply:
done